flow/shared/postgres/type_conversion.go

```diff
@@ -133,6 +133,8 @@ func CustomTypeToQKind(typeData shared.CustomDataType) types.QValueKind {
 		return types.QValueKindGeography
 	case "hstore":
 		return types.QValueKindHStore
+	case "vector", "halfvec", "sparsevec":
```

Comment:
sparsevec might better be passed along as Map(Int32, Float32) but that would require a new QValueKind